wafer/talks/templates/wafer.talks/talk.html

```diff
@@ -41,6 +41,13 @@
 <div class="well">
     {{ object.abstract.rendered|safe }}
 </div>
+{% if perms.edit_private_notes and object.private_notes %}
+<h2>Private notes</h2>
+<p>(The following is not visible to submitters or attendees.)</p>
+<div class="well">
+  {{ object.private_notes|urlize|linebreaks }}
```

Comment:
I now made it configurable via `settings.py` but left the default at `True` (i.e. show private notes). I am not wed to this default setting.